server/authentication/index.js

```diff
@@ -14,13 +14,7 @@ function authenticationMiddleware() {
   };
 }
 
-passport.serializeUser(({ forename, surname, username, email, authorities }, done) => done(null, {
-  forename,
-  surname,
-  username,
-  email,
-  authorities,
-}));
+passport.serializeUser((user, done) => done(null, user));
```

Comment:
IIRC this was like this so that the password didn't get sent to front end

Reply:
Good catch!!